### ISSUE TITLE
[TABS] Fix alignment on tabs

### ISSUE THREAD

**Issue by tiagoalvesdulce** [tiagoalvesdulce/pi-ui#146]
Alignment on vertical tabs should be `flex-start` but we should keep `flex-end` for horizontal tabs.

**Comment by fernandoabolafio** [comment]
@tiagoalvesdulce can you provide more details on it? I couldn't notice anything wrong with the tabs.

**Comment by tiagoalvesdulce** [comment]
I refer to the tabs when they become a dropdown for responsivity. They are currently aligned to the right (flex-end).

**Comment by amassarwi** [comment]
@tiagoalvesdulce Do you mean the `dropdown` mode for tabs we use for mobile tabs in politeiagui ?
We already aligned the active tab with dropdown list here (https://github.com/tiagoalvesdulce/pi-ui/pull/144) but it seems like on master it's broken after latest changes:
![image](https://user-images.githubusercontent.com/10324528/64922401-24f8d480-d7cf-11e9-8ffc-792627d71803.png)
Also in `pi-ui` docs:
![image](https://user-images.githubusercontent.com/10324528/64922422-4e196500-d7cf-11e9-8bb0-f6f827f65e46.png)

**Comment by tiagoalvesdulce** [comment]
Exactly. On pigui we should place a `flex-end` alignment to the Tabs container in order for them to go down and be placed correctly. When we do that, the dropdown mode gets messed up. This shouldn't happen.

**Comment by amassarwi** [comment]
Checking...
